contract/build.gradle.kts

```diff
@@ -6,11 +6,15 @@ buildscript {
 
 plugins {
     id("kotlin")
-
 }
 
 dependencies {
     api(project(":proto"))
-    implementation("io.provenance.p8e:p8e-contract-base:0.8.+")
-    implementation("io.provenance.model:metadata-asset-model:0.1.2")
+    implementation("io.provenance.scope:contract-base:0.4.9")
+    implementation("io.provenance.scope:util:0.4.9")
+    implementation("io.provenance.model:metadata-asset-model:loan-wrapper") {
```

Comment:
Might want a todo here as well so we can change this once https://github.com/provenance-io/metadata-asset-model/pull/6 is checked in